.php_cs

```diff
@@ -1,11 +1,23 @@
 <?php
 
+$header = <<<EOF
+This file is part of the PHP CS utility.
```

Comment:
I think this is the correct header. The change from `Symfony CS` to `PHP CS` was introduced by @fabpot in 926ad7d